heft-plugins/heft-sass-plugin/src/schemas/heft-sass-plugin.schema.json

```diff
@@ -46,7 +55,7 @@
       "description": "A list of paths used when resolving Sass imports.",
       "items": {
         "type": "string",
-        "pattern": "[^\\\\]"
+        "pattern": "[^\\\\\\/]"
```

Comment:
Disallowing all folder separators?

Reply:
Intent was to forbid absolute paths. Revising.